lib/defaults-utils.js

```diff
@@ -12,7 +12,7 @@ import log from './logger';
  * @param {any} value The value to be serialized
  * @param {boolean} serialize [true] Whether to serialize the resulting
  * XML to string or to return raw HTMLElement instance
- * @returns {Node|string} Either string or raw node representation of
+ * @returns {Element|string} Either string or raw node representation of
```

Comment:
it must not be imported explicitly if only used for typing. Use `import(...).Element` syntax instead